lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -160,12 +160,6 @@ void NativeCodeGenerator::Close()
     // Close FreeLoopBodyJobManager first, as it depends on NativeCodeGenerator to be open before it's removed
     this->freeLoopBodyManager.Close();
 
-    // Remove only if it is not updated in the debug mode (and which goes to interpreter mode).
-    if (!hasUpdatedQForDebugMode || Js::Configuration::Global.EnableJitInDebugMode())
-    {
-        Processor()->RemoveManager(this);
```

Comment:
Should keep: `Processor()->RemoveManager(this);` just get rid of the `if` around it. (We're removing this flag because it's always true - hence this was an if(true).